packages/cli/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Change Log
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Added a warning to users using a TSO command. It now warns them that a logon procedure is not being used when a non default logon procedure
+and `--ssm` option is used.[#2528](https://github.com/zowe/zowe-explorer-vscode/issues/2528)
```

Comment:
```suggestion
- BugFix: Added a warning to users using a TSO command. It now warns them that a logon procedure is not being used when a non default logon procedure and `--ssm` option is used.[#2528](https://github.com/zowe/zowe-explorer-vscode/issues/2528)
```

no need for another commit 😋
this can be fixed on any other PR 🙏

Reply:
thanks for fixing the unit test 🙏